exonum/src/messages/signed.rs

```diff
@@ -237,6 +238,15 @@ where
     }
 }
 
+impl<T: Debug> Debug for Verified<T> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Verified")
+            .field("raw", &self.raw.to_string())
```

Comment:
Could it make sense to output the message author instead of a complete `raw` message?

Reply:
Yes this is more useful information for debugging.